apps/omg/test/omg/state_test.exs

```diff
@@ -51,7 +51,8 @@ defmodule OMG.StateTest do
   @tag fixtures: [:alice, :standalone_state_server]
   test "can execute various calls on OMG.State, one happy path only", %{alice: alice} do
```

Comment:
I'm having difficulties understanding the purpose of this test... can't see why your changes would affect the test.

Reply:
Ouch left off from initial branch - reverted.
Thx!